site/docs/v1.2.0/customize-installation.md

```diff
@@ -63,17 +63,17 @@ To use a different volume snapshot provider:
 
 1. Add your volume snapshot provider's plugin to Velero (look in [your provider][0]'s documentation for the image name):
 
-    ```bash
-    velero plugin add <registry/image:version>
-    ```
+   ```bash
```

Comment:
could you revert this whitespace change? It causes things to render poorly on the website (see the preview at https://deploy-preview-2209--velero.netlify.com/docs/v1.2.0/customize-installation/ -- note code blocks no longer indented and numbering restarts). You'll need to do the same on the `master` version of this page too.